README.md

```diff
@@ -452,6 +452,27 @@ new command, add it to `_processCommand`, write and call processor for it.
 
 ### Publish new `npm` release
 
+#### Release branches
+
+`chromium-bidi` maintains release branches corresponding to Chrome releases. The
+branches are named using the following pattern: `releases/m$MAJOR_VERSION`.
+
+The new release branch is created as soon a new major browser version is
+published by the
+[update-browser-version](https://github.com/GoogleChromeLabs/chromium-bidi/blob/main/.github/workflows/update-browser-version.yml)
+job:
+
+- the PR created by this job should be marked as a feature and it should cause the
```

Comment:
@Lightning00Blade WDYT? I think it is the only part that we need to implement right now and the rest of the workflow we can start doing if we need it.